appveyor.yml

```diff
@@ -27,6 +27,9 @@ install:
   - edm install --version 3.6 -y numpy nose mock Sphinx coverage psutil
   - "%CMD_IN_ENV% edm run -- pip install -r ci\\ci-requirements.txt"
   - edm run -- pip install vtk pyqt5==5.9.2
+  # get a sufficiently modern OpenGL for VTK9
+  - git clone --depth 1 git://github.com/pyvista/gl-ci-helpers.git
+  - powershell gl-ci-helpers/appveyor/install_opengl.ps1
```

Comment:
Here we just download and install a Mesa OpenGL lib with OpenGL 3.3+ support. I originally made this for VisPy, then repackaged it once PyVista wanted to use it. It has been stable across those repos for months/years now.

Reply:
You can see in the [AppVeyor log](https://ci.appveyor.com/project/EnthoughtOSS/mayavi/builds/33689968) before this commit complaints about rendering capabilities:
```
Tests if TVTK scene can be garbage collected. ... 2020-06-23 14:11:13.868 (   3.287s) [                ]vtkWin32OpenGLRenderWin:635    ERR| vtkWin32OpenGLRenderWindow (00000008BF96D010): failed to get wglChoosePixelFormatARB
2020-06-23 14:11:13.884 (   3.303s) [                ]vtkWin32OpenGLRenderWin:713    ERR| vtkWin32OpenGLRenderWindow (00000008BF96D010): failed to get valid pixel format.
2020-06-23 14:11:13.887 (   3.306s) [                ]vtkOpenGLRenderWindow.c:569    ERR| vtkWin32OpenGLRenderWindow (00000008BF96D010): GLEW could not be initialized: Missing GL version
```

Reply:
This is very useful thanks!